README.md

```diff
@@ -43,10 +43,10 @@ If you want a simple guide to install Empirical Core, then you've come to the ri
 
 9. Run Redis with ```redis-server```
 
-11. Run npm installer with ```npm install && cd ./client && npm install```
```

Comment:
Looks like this will be cleared up in the merge, as your fork has a non-rebased branch (but no conflicts in README.md helpfully).